examples/hosts/app-integration/external-controller/src/app.ts

```diff
@@ -3,15 +3,14 @@
  * Licensed under the MIT License.
  */
 import { KeyValueDataObject } from "@fluid-experimental/data-objects";
-import Fluid from "@fluid-experimental/fluid-static";
-import { TinyliciousService } from "@fluid-experimental/get-container";
+import { FluidTinylicious } from "@fluid-experimental/fluid-static";
+import { ITinyliciousFileConfig } from "@fluid-experimental/get-container";
```

Comment:
This should be coming from the same package as the Tinylicious client bits. Let's consolidate the two packages. More comments below.

Reply:
`external-controller` only requires an import from `@fluid-experimental/tinylicious-client` in the follow-up https://github.com/microsoft/FluidFramework/pull/5838